torch/_decomp/__init__.py

```diff
@@ -299,6 +302,10 @@ def _is_cia_op(op: "OpOverload") -> bool:
     )
 
 
+def _is_preservable_cia_op(op: "OperatorBase") -> bool:
+    return _check_valid_to_preserve(op) and _is_cia_op(op)
```

Comment:
tbh, `_collect_all_valid_cia_ops` and its related helpers could probably all go in `exported_program.py` instead of needing to live here with the other decomp main API's

Reply:
Yeah hahha good idea! I will move them.

Reply:
Hmm actually i think it is bit hard because we do need this APIs to create the new core_aten_decomp table. Can I follow up on it later after I move all core_aten_decomp references to export alias of core_aten_decomp? Then we will be doing this compute in exported_program.py.